pkg/model/stack_serializer_test.go

```diff
@@ -2286,3 +2286,67 @@ func Test_TranslateOktetoStackPortsToComposePorts(t *testing.T) {
 		})
 	}
 }
+
+func Test_setLivenessFromDependsOnServiceHealthy(t *testing.T) {
+	type args struct {
+		svc     *Service
+		s       *Stack
+		svcName string
+	}
+	tests := []struct {
+		args args
+		want *Service
+		name string
+	}{
+		{
+			name: "app depends_on service_healthy db",
+			args: args{
+				svc:     &Service{},
+				svcName: "db",
+				s: &Stack{
+					Services: map[string]*Service{
+						"app": {
+							DependsOn: DependsOn{
+								"db": {
+									Condition: "service_healthy",
```

Comment:
NIT: Should we use the constant here? If the name changes we don't need to change it in the tests